### PR TITLE
RET-1842: ET1 Vetting - Jurisdiction codes

### PR DESCRIPTION
**Pull request by robin-ieong**
(https://tools.hmcts.net/jira/browse/RET-1842
https://tools.hmcts.net/jira/browse/RET-1843)

### Change description ###
This PR contains the implementation of both RET-1842 and RET-1843, please see tickets.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by nickwest1 — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -5993,6 +5993,762 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "existingJurisdictionCodes",
+    "UserRole": "caseworker-employment-etjudge",
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:5999]: Change UseRole access for caseworker-employment-etjudge and caseworker-employment to R for all fields.

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]
Change UseRole access for caseworker-employment-etjudge and caseworker-employment to R for all fields.

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -8183,6 +8171,762 @@
...
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "existingJurisdictionCodes",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "existingJurisdictionCodes",
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:8194]: This is a readonly user role.

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: definitions/json/AuthorisationCaseField.json (hunk 2 above)
Comment (Itay-Kirsh) [reply definitions/json/AuthorisationCaseField.json:8194]: There's many other examples, I only pointed out one.

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**